slurm/train.slurm

```diff
@@ -32,6 +32,12 @@ WORLD_SIZE=$(($NUM_NODES*$GPUS_PER_NODE))
 # Due to conflicts between Accelerate's DeepSpeed configs and Transformers' TrainingArguments, we need to parse the gradient accumulation steps from the config file to ensure they match
 CONFIG_FILE=recipes/$MODEL/$TASK/config_$CONFIG_SUFFIX.yaml
 GRAD_ACC_STEPS=$(grep 'gradient_accumulation_steps' $CONFIG_FILE | awk '{print $2}')
+USE_VLLM=$(grep 'use_vllm:\s*true' $CONFIG_FILE) # Match "use_vllm: true" (with optional whitespace)
+
+if [ -n "$USE_VLLM" ]; then  # Check if USE_VLLM is *not* empty (found)
+    NUM_PROCESSES=7
```

Comment:
Since WORLD_SIZE isn't used anywhere else, how about we do this:

```
if [ -n "$USE_VLLM" ]; then
    WORLD_SIZE=$(($WORLD_SIZE-1))
fi
```

That way it also works if one wants to train on different setups (e.g. 3+1 GPUS)